### DISCUSSION TITLE
Forsaken refugees  in Orgrimmar - is it 335a or 3.4.0?

### DISCUSSION THREAD

**Discussion by wowquester** [azerothcore/azerothcore-wotlk#16858] [General]
I have just noticed there are many forsaken refugees, shelters and so on... that I've never seen before. 
Is it really 335a content? Wowhead says they were added in 340.
Just asking, correct me if Iam wrong.

**Comment by heyitsbench** [comment]
Are you in GM mode when you see these creatures? They are involved in the Battle for UC questline phasing, and GM mode can view all creatures in each phase. These creatures are 3.3.5a.
